recipes/ncbi-cxx-toolkit-public/26.0/conanfile.py

```diff
@@ -0,0 +1,1587 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanException
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+import subprocess
+import tempfile
+import sys
+import stat
+
+def _remove_readonly(fn, path, excinfo):
+    try:
+        os.chmod(path, stat.S_IWRITE)
+        fn(path)
+    except Exception as exc:
+        print("Skipped: ", path, " because ", exc)
+
+class NcbiCxxToolkit(ConanFile):
+    name = "ncbi-cxx-toolkit-public"
+    license = "CC0-1.0"
+    homepage = "https://ncbi.github.io/cxx-toolkit"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "NCBI C++ Toolkit"
+    topics = ("ncbi", "biotechnology", "bioinformatics", "genbank", "gene",
+              "genome", "genetic", "sequence", "alignment", "blast",
+              "biological", "toolkit", "c++")
+    settings = "os", "compiler", "build_type", "arch"
+
+    tk_tmp_tree = ""
+    tk_src_tree = ""
+    all_NCBI_requires = {}
+
+    options = {
+        "shared":     [True, False],
+        "sharedDeps": [True, False],
```

Comment:
If there are vendorized deps we should try to package them first, before adding this recipe.